react-troubleshooter.md

```diff
@@ -4,6 +4,16 @@
 
 ### __invariant violation__:
 
+#### Unable to find element
+
+When the DOM has mutated & you try to modify props / state on a component, you will find that React will complain about not being able to find the element.
```

Comment:
This issue only seems to happen when the DOM that React has "bound to" (for want of a better way to describe it) has gone. This would also apply to parent nodes as the child would be removed at the same time the parent is removed. Essentially, if you change a thing & React has no idea what you've done, it throws this error.